src/main/java/seedu/duke/Duke.java

```diff
@@ -67,7 +73,7 @@ public String readCommand(){
      */
     private CommandResult executeCommand(Command command) {
         try {
-            CommandResult result = command.execute();
+            CommandResult result = command.execute(items,myBudget);
```

Comment:
Actually can just use command.execute() without any arguments. Can refer to my PR especially in the Command parent class where I included protected variables for ShoppingList and Budget and have a method called setData which initialises the shoppingList and the budget.

Reply:
Ok, I'll include the change when I'm resolving the conflicts, thanks!